data/utils/defectsFromBias.py

```diff
@@ -112,13 +46,20 @@ def writeDefectsFile(bboxList, path, detectorSerial, detectorName):
     args = parser.parse_args()
 
     biasMI = afwImage.MaskedImageF(args.bias)
-    defectList = getDefectListFromMask(biasMI, "BAD")
-    bboxList = [defect.getBBox() for defect in defectList]
-    writeDefectsFile(bboxList, DefectsPath, detectorSerial, detectorName)
+    defectList = Defects.fromMask(biasMI, "BAD")
```

Comment:
Please consider changing the description string at line 39 to an f string, for clarity.